tox.ini

```diff
@@ -18,9 +18,7 @@ changedir = docs
 commands = sphinx-build -W -b html -d {envtmpdir}/doctrees . {envtmpdir}/html
 
 [testenv:lint]
-deps =
```

Comment:
This should stay here, so that the lint env does not download all of mitmproxy's dependencies/installs mitmproxy itself.